plugins/woocommerce/tests/e2e-pw/tests/product/product-variable.spec.js

```diff
@@ -317,6 +317,8 @@ test.describe(
 
 			await page.locator( '#colour' ).selectOption( 'Red' );
 
+			await page.waitForTimeout( 300 );
+
```

Comment:
Some tests find the `.woocommerce-variation-price` element to get its text content once a product variation is selected. However, they started to fail after implementing the delay of 300 ms before updating the DOM.

The solution was to add `await page.waitForTimeout( 300 );` right after every time the product variation is selected.